src/components/map/FlowText.js

```diff
@@ -51,6 +51,7 @@ const FlowText = props => {
 					textAnchor="start"
 					fill={mapStyleDefs.link.flowText}
 					text={link.flowValue}
+					styles={mapStyleDefs}
```

Comment:
Im not sure how the website works this out, but styles is certainly used internally to componentTextSymbol, so I have converted this over and passed it through.